lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py

```diff
@@ -150,12 +150,19 @@ def primNameTextChanged(primName):
         if validatedName != primName:
             cmds.textFieldGrp('primNameText', edit=True, text=validatedName)
 
+def variantOrNewPrim(buttonChecked):
+    # The variant and new child prim radio buttons are part of a collection.
+    # So only one can be checked.
+    variantSelected = cmds.radioButtonGrp('variantRadioBtn', query=True, select=1)
+    cmds.rowLayout('usdCacheVariantSetRow', edit=True, enable=variantSelected)
+    cmds.rowLayout('usdCacheVariantNameRow', edit=True, enable=variantSelected)
+    cmds.textFieldGrp('primNameText', edit=True, enable=not variantSelected)
```

Comment:
Enable or disable certain sections based on whether the variant or new prim radio button is selected.